src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -143,6 +143,10 @@ public function getFlashes(string|array $types = null): array
             return [];
         }
 
+        if (!method_exists($session, 'getFlashBag')) {
```

Comment:
You can add a comment that this can be updated in 7.0 to check `FlashBagAwareSessionInterface`, when `symfony/http-foundation: 6.4` is required.